core/examples/luigi-sample-angular/src/index.html

```diff
@@ -2,6 +2,7 @@
 <html>
 <head>
   <title>Luigi</title>
+  <link rel='stylesheet' href='/luigi-core/fundamental-ui/fundamental-ui.min.css'>
```

Comment:
I think it is a good move to import fundamental-ui this way, and in the future consolidate it inside luigi.css. Removes complexity regarding the usage of fundamental-ui. Awesome!
I have done a quick check and after importing fundamental-ui this way looks like we can remove all imports for fundamental-ui from Luigi Core. Could you also please check on your side and remove them if possible?